### ISSUE TITLE
Recommendation 20 API paths

### ISSUE THREAD

**Issue by kshychko** [edi3/edi3-codelists#10]
This issue is to kick off a discussion on the question if the codelist API for the recommendation 20 shall specify different paths for Annex I, Annex II and Annex III.

```
The three annexes are structured accordingly:
Annex I – Code elements listed by quantity category. This annex is normative and
contains only level 1 and 2 entries.
Annex II – Code elements listed by name. This annex is informative and contains all
level 1, 2 and 3 entries.
Annex III – Code elements listed by common code. This annex is informative and
contains all level 1, 2 and 3 entries.
```

Considering there are different datasets at the moment, that might make sense. 
So at the moment we have quantityCategories for Annex I, unitsOfMeasure for Annex || and we a missing path for Annex III yet.

From other perspective it is better to maintain a single data source, so probably a combination of normative and information data at all levels entries shall be provided as a single path with different ways to query it - by quantity category, name or common code.

The third option would be specify the informative dataset with all levels entries, if it's a real business case and there would be no use in normative smaller dataset.

@cmsdroff , @rgenoulaz, @Lewis-David-Wong please let me know if my understanding is correct and share your thoughts.

**Comment by rgenoulaz** [comment]
I definitely prefer the second option.
From a restful perspetive this is the same resource, it's only a pity that we could not unify the two files with several levels of details embeded.

**Comment by onthebreeze** [comment]
I Agree - it's one resource.  Why cant we combine annex I and annex II into one list?  I cant see any reason why not?
